src/main/java/exceptions/AuthorDoesNotExistException.java

```diff
@@ -0,0 +1,12 @@
+package exceptions;
+
+public class AuthorDoesNotExistException extends TantouException {
+    public AuthorDoesNotExistException(String authorName) {
+        super(generateAuthorDoesNotExistMessage(authorName));
+    }
+
+    private static String generateAuthorDoesNotExistMessage(String authorName) {
+        return "Author [" + authorName + "] does not exist! "
+                + "Author needs to exist in our catalog first before adding their sales data!";
+    }
```

Comment:
I appreciate the custom exception for this particular invalid input, but I think we should make this message more generic in case other `Command` classes want to use it.

Reply:
Fixed in #106